packages/lab/src/dialog/Dialog.tsx

```diff
@@ -55,27 +68,39 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(function Dialog(
   const {
     children,
     className,
-    open = true,
+    open = false,
     onOpenChange,
     status,
-    initialFocus,
+    id,
+    closeOnBlur = true,
```

Comment:
The dialog doesn't receive focus therefore doesn't get blurred, we should update this name

Reply:
change to `disableDismiss`